docs/getting_started/current-sem-trainings.md

```diff
@@ -56,6 +57,8 @@ Click the name of a training to view more details and register!
 ### **AI/ML**
 - [Setting up LLMs on CURC Resources (an RC Short Course)](https://colorado.libcal.com/calendar/events/llm) - **September 26th, 11am, Virtual (Zoom)**
 - [Applied Containerization for Machine Learning in HPC (an RC Short Course)](https://colorado.libcal.com/calendar/events/contain) - **October 21st, 10am, Virtual (Zoom)**
+### **Cloud**
```

Comment:
Do we want to create a new heading for this? I am wondering if it would be better to put this under the AI/ML section. From the title and description, the thing being discussed here is AI. Although I do understand that it uses cloud resources. If you feel strongly about putting this under a new section, I am fine with it. I will approve either way.